pkg/moq/template.go

```diff
@@ -83,20 +83,10 @@ func (mock *{{$obj.InterfaceName}}Mock) {{.Name}}({{.Arglist}}) {{.ReturnArglist
 // {{.Name}}Calls gets all the calls that were made to {{.Name}}.
 // Check the length with:
 //     len(mocked{{$obj.InterfaceName}}.{{.Name}}Calls())
-func (mock *{{$obj.InterfaceName}}Mock) {{.Name}}Calls() []struct {
-		{{- range .Params }}
-		{{ .Name | Exported }} {{ .Type }}
-		{{- end }}
-	} {
-	var calls []struct {
-		{{- range .Params }}
-		{{ .Name | Exported }} {{ .Type }}
-		{{- end }}
-	}
+func (mock *{{$obj.InterfaceName}}Mock) {{.Name}}Calls() []{{.Name}}Call {
 	lock{{$obj.InterfaceName}}Mock{{.Name}}.RLock()
-	calls = mock.calls.{{.Name}}
-	lock{{$obj.InterfaceName}}Mock{{.Name}}.RUnlock()
-	return calls
+	defer lock{{$obj.InterfaceName}}Mock{{.Name}}.RUnlock()
```

Comment:
Could we have this on two lines? I'm not a fan of compressing code onto one line at the cost of readability?